gc/base/standard/HeapWalker.cpp

```diff
@@ -93,8 +94,10 @@ heapWalkerObjectSlotDo(OMR_VMThread *omrVMThread, MM_HeapRegionDescriptor *regio
 	 * OK. We should simplify this code once classes-on-heap is enabled
```

Comment:
Can you clean up the comment while you are making changes to remove the references to classes and talk about other off heap memory or fields that need to be updated?  What if there is more than one indirect reference from the object?